api/V1/util.js

```diff
@@ -88,22 +88,23 @@ function addRecordingFromPost(model, request, response) {
 }
 
 function getRecordingFile(modelClass, request, response) {
-
   // Check that if the request was authenticated that is was
   // authenticated by a user JWT, not a device JWT.
   // TODO get passport to do this...
-  if (request.user !== null && !requestUtil.isFromAUser(request))
-  {return responseUtil.notFromAUser(response);}
+  if (request.user !== null && !requestUtil.isFromAUser(request)) {
+    return responseUtil.notFromAUser(response);
```

Comment:
Can put the return statement on a new line like you have below.

Reply:
Will do.